frontend/components/ProfilePublication/Publication.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+
+class Publication extends Component {
+  render() {
+    return (
+      <>
+        {/* {style} */}
+        <a
+          href="{this.props.data.link}"
+          style={{
+            fontSize: 17,
```

Comment:
I don't actually recommend using pixel (this is the default if just a number is present) as it is hard to make it responsive in design. Use either _%_ or even better _em_ with fonts

Reply:
For me its pretty open they all have their benefits. Sometimes you just want to font to stay the same across all sizes.